source/llassetgen/include/llassetgen/FntWriter.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <llassetgen/Image.h>
+
+#include <ft2build.h>
+#include FT_FREETYPE_H
+
+namespace llassetgen {
+	struct Info {
+		std::string face;
+		int size;
+		bool is_bold;
+		bool is_italic;
+		std::string charset;
+		bool use_unicode;
+		/*
+		float stretch_h;
```

Comment:
Friendly reminder: We agreed on camelCase for our variables. (I know that some c++ and freetype variables use underscore, but we already discussed that.)